src/normalize-options.js

```diff
@@ -71,7 +69,7 @@ export const validateModulesOption = (modulesOpt = "commonjs") => {
 
 export default function normalizeOptions(opts) {
   // TODO: remove whitelist in favor of include in next major
-  if (opts.whitelist && !hasBeenWarned) {
+  if (opts.whitelist) {
     console.warn(
```

Comment:
then we should add hasBeenWarned = true in the if block? otherwise it will warn multiple times?